site/content/docs/main/csi-snapshot-data-movement.md

```diff
@@ -121,6 +121,24 @@ oc annotate namespace <velero namespace> openshift.io/node-selector=""
 oc create -n <velero namespace> -f ds.yaml
 ```
 
+**OpenShift on IBM Cloud**
+
+
+Update the host path for volumes in the node-agent DaemonSet in the Velero namespace from `/var/lib/kubelet/pods` to
+`/var/data/kubelet/pods`.
+
+```yaml
+hostPath:
+  path: /var/lib/kubelet/pods
+```
+
+to
+
+```yaml
+hostPath:
+  path: /var/data/kubet/pods
```

Comment:
@msfrucht Shouldn't this be `kubelet` ? (`s/kubet/kubelet`)

Reply:
Yes. My error. I'll get that fixed.

Reply:
I believe I've resolved the error and looked over the changes again to check for any other issues.